README.md

```diff
@@ -74,3 +74,8 @@ if appErr := p.API.SetProfileImage(userID, profileImage); appErr != nil {
 
 ### How do I build the plugin with unminified JavaScript?
 Use `make debug-dist` and `make debug-deploy` in place of `make dist` and `make deploy` to configure webpack to generate unminified Javascript.
+
+### How do I run the tests from IntelliJ IDE?
+
+To run the tests from the IDE you need to enable the setting `Enable go Modules Integration`. 
```

Comment:
Maybe we place it under https://developers.mattermost.com/contribute/ instead? This is not specific to this repo. @lieut-data @pradeepmurugesan Thoughts?

Reply:
I feel it's better here because, developers try use this template to build their own plugin. Not sure if they will search this issue under https://developers.mattermost.com/contribute/.

Reply:
My concerns is that this Readme will become a depot for information about different IDE's and systems. What do you think about adding this information to https://developers.mattermost.com/contribute/ and linking to this page in the readme?

Reply:
As an alternative, is there any opportunity to provide an IntelliJ template for plugin authors to consume from this repository?

Reply:
@pradeepmurugesa What do you think about these ideas?